README.md

```diff
@@ -10,102 +8,72 @@ SurveyJS PDF exporter library is an easy way to render [SurveyJS Library](https:
 <img alt="Closed issues" title="Closed Issues" src="https://img.shields.io/github/issues-closed/surveyjs/survey-pdf.svg">
 </a>
 
+SurveyJS PDF Export is a client-side extension over the [SurveyJS Library](https://github.com/surveyjs/survey-library) that enables users to save surveys as PDF documents.
```

Comment:
SurveyJS PDF Export is a client-side extension over the [SurveyJS Library](https://github.com/surveyjs/survey-library), which enables users to save surveys as PDF documents.